### PR TITLE
Housekeeping Update Packages

### PR DESCRIPTION
**Pull request by ChrisPulman**
Dependabot failing to update due to multiple dependencies

**What kind of change does this PR introduce?**

Update

**What is the current behavior?**

Dependabot is failing to update multiple packages with dependencies.

**What is the new behavior?**

Bulk update packages to resolve conflicts.

**What might this PR break?**

none

**Please check if the PR fulfills these requirements**
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**Other information**:

### CODE REVIEW

File: src/ReactiveUI.AndroidX/ReactiveUI.AndroidX.csproj [hunk 1]
```diff
@@ -16,11 +16,11 @@
...
   <ItemGroup>
-    <PackageReference Include="Xamarin.AndroidX.Core" Version="1.12.0.1" />
-    <PackageReference Include="Xamarin.AndroidX.Preference" Version="1.2.1.1" />
-    <PackageReference Include="Xamarin.AndroidX.Legacy.Support.Core.UI" Version="1.0.0.21" />
-    <PackageReference Include="Xamarin.Google.Android.Material" Version="1.9.0.3" />
-    <PackageReference Include="Xamarin.AndroidX.Lifecycle.LiveData" Version="2.6.2.1" />
+    <PackageReference Include="Xamarin.AndroidX.Core" Version="1.12.0.2" />
+    <PackageReference Include="Xamarin.AndroidX.Preference" Version="1.2.1.2" />
+    <PackageReference Include="Xamarin.AndroidX.Legacy.Support.Core.UI" Version="1.0.0.22" />
+    <PackageReference Include="Xamarin.Google.Android.Material" Version="1.10.0.1" />
```
Comment (glennawatson) [inline src/ReactiveUI.AndroidX/ReactiveUI.AndroidX.csproj:22]: Sometimes this stuff can cause problem if updated in my experience with the Xamarin material packages etc.

Comment (ChrisPulman) [reply src/ReactiveUI.AndroidX/ReactiveUI.AndroidX.csproj:22]: So should we fix the versions? and tell dependabot to ignore them.

Comment (glennawatson) [reply src/ReactiveUI.AndroidX/ReactiveUI.AndroidX.csproj:22]: Yeah maybe, we aren't supporting Xamarin beyond middle of next year anyway with Microsoft stopping support anyway.

Comment (ChrisPulman) [reply src/ReactiveUI.AndroidX/ReactiveUI.AndroidX.csproj]: Reverted

**Review by glennawatson — APPROVED** [review]

**Pull request merged**